src/Compilers/CSharp/Portable/Compilation/AwaitExpressionInfo.cs

```diff
@@ -1,33 +1,27 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
     /// <summary>
-    /// Structure containing all semantic information about an await expression.
+    /// Public structure containing all semantic information about an await expression.
     /// </summary>
     public struct AwaitExpressionInfo : IEquatable<AwaitExpressionInfo>
```

Comment:
>AwaitExpressionInfo [](start = 18, length = 19)

Are we adding SemanticModel API similar to GetAwaitExpressionInfo?

Reply:
I'm not sure yet. Added a work item to list: https://github.com/dotnet/roslyn/issues/24037